### PR TITLE
trying to improve the login email

### PR DESCRIPTION
**Pull request by KevinSayers**
Attempt at #114 trying to make it a bit more attention grabbing. 

Needs a higher quality nf-tower logo, and to put both the Seqera and nf-tower images in this repo.

### CODE REVIEW

File: tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html [hunk 1]
```diff
@@ -52,26 +52,29 @@
     <tr>
         <td style="font-family:'-apple-system', BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';font-size:16px;vertical-align:top;" valign="top"></td>
         <td class="container" style="font-family:'-apple-system', BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';font-size:16px;vertical-align:top;display:block;margin:0 auto !important;max-width:580px;padding:30px 30px 0px 30px;width:580px;" width="580" valign="top">
+            <img src="https://raw.githubusercontent.com/seqeralabs/nf-tower/master/tower-web/src/assets/landing/assets/img/nf-tower-purple.svg?sanitize=true" alt="nf tower logo">
             <div class="content" style="display:block;margin:0 auto;max-width:580px;">
```
Comment (pditommaso) [inline tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html]: Images should be encoded, other anti-spam can block it. We decided to put only in the welcome email. I would keep this without any image.

File: tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html [hunk 2]
```diff
@@ -52,26 +52,29 @@
...
                 <p style="font-family:'-apple-system', BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';font-size:16px;font-weight:normal;line-height:1.5;margin:0;margin-bottom:15px;">
-                    Dear ${user},<br>
-                    Please complete the sign in clicking the button below:
+                    <br> ${user}, welcome to Tower!<br>
+                    <br>
+                    Complete your sign in to Nextflow Tower by clicking the button below and get ready to monitor your workflows like never before with seamless
+                    Nextflow integration. 
                 </p>
```
Comment (pditommaso) [inline tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html]: Reads good! what about *.. and start to monitor* ?

Comment (KevinSayers) [reply tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html]: Sounds good, I changed to _start monitoring_

File: tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html (hunk 1 above)
Comment (KevinSayers) [reply tower-backend/src/main/resources/io/seqera/tower/service/auth-mail.html]: Removed the images.

**Comment by pditommaso** [comment]
I've merged this manually. Thanks!

**Pull request closed without merging**